core/base/module.php

```diff
@@ -301,4 +303,25 @@ protected function get_assets_base_url() {
 	protected function get_assets_relative_url() {
 		return 'assets/';
 	}
+
+	/**
+	 * Initialize the module related widgets.
+	 */
+	public function init_widgets() {
+		if ( ! method_exists( $this, 'get_widgets' ) ) {
```

Comment:
1. Implement it at the base module and return an empty array
2. Add a task to remove the related methods from the Pro module